migrations/2_dev_dependencies.js

```diff
@@ -1,7 +1,7 @@
 const Artifactor = require("@truffle/artifactor")
 const migrateBatchExchange = require("@gnosis.pm/dex-contracts/src/migration/PoC_dfusion")
 const { GnosisSafe } = require("../scripts/utils/dependencies")(web3, artifacts)
-const ProxyFactory = artifacts.require("./GnosisSafeProxyFactory.sol")
+const { GnosisSafeProxyFactory } = require("../scripts/utils/dependencies")(web3, artifacts)
```

Comment:
Can these both be imported in the same line?

Reply:
Yes they can...